de.fu_berlin.inf.dpp.eclipse/META-INF/MANIFEST.MF

```diff
@@ -1,7 +1,7 @@
 Manifest-Version: 1.0
 Bundle-ManifestVersion: 2
 Bundle-Name: Saros Plug-in
-Bundle-SymbolicName: de.fu_berlin.inf.dpp;singleton:=true
+Bundle-SymbolicName: de.fu_berlin.inf.dpp.eclipse;singleton:=true
```

Comment:
Yes, there will be follow up refactoring changes:
The second refactoring will rename the prefix `de.fu_berlin.inf.dpp` to `saros` (in directories and packages).
The third refactoring will create a package `saros.eclipse` which leads to a package structure that is equal to the bundle name.
I want to split this into two refactoring pull requests, because I want to separate changes which are only renaming and changes that modify the project structure.